.ci/containers/contributor-checker/check-contributor.sh

```diff
@@ -13,12 +13,12 @@ set -x
 ASSIGNEE=$(curl -H "Authorization: token ${GITHUB_TOKEN}" \
   "https://api.github.com/repos/GoogleCloudPlatform/magic-modules/pulls/${PR_NUMBER}/requested_reviewers" | jq .users[0].login)
   
-if [ "$ASSIGNEE" == "null" || -z "$ASSIGNEE" ]; then 
+if [ "$ASSIGNEE" == "null" ] || [ -z "$ASSIGNEE" ] ; then 
```

Comment:
```suggestion
if [[ "$ASSIGNEE" == "null" || -z "$ASSIGNEE" ]] ; then 
```

as per http://google.github.io/styleguide/shellguide.html#s6.3-tests